WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -1411,6 +1411,10 @@ - (void)savePost
     [self logSavePostStats];
 
     [self.view endEditing:YES];
+
+    //Make sure that we are uploading the latest content on the editor.
+    self.post.postTitle = self.titleText;
+    self.post.content = self.bodyText;
```

Comment:
Should we also call `[self.post save];` to ensure that the change is persisted to core data, or maybe just call `[self autosaveContent]` here which does the same work *and* saves to core data.